ansible/roles/system/tasks/main.yml

```diff
@@ -68,6 +68,36 @@
   tags:
     - touches_boot_partition
 
+- name: Comment out the original dtoverlay config.
+  replace:
+    dest: ./config.txt
+    regexp: ^(dtoverlay=vc4-kms-v3d)$
+    replace: '#\1'
+
+- name: Change dtoverlay value for DRM VC4 V3D driver.
+  replace:
+    dest: ./config.txt
```

Comment:
This needs to be `/boot/config.txt`.

Reply:
Thanks! I forgot to change it to an absolute path. Was experimenting with Ansible file manipulation on a separate repo.